source/configure/site-configuration-settings.rst

```diff
@@ -1151,33 +1151,35 @@ Enable message link previews
   :configjson: .ServiceSettings.EnableSVGs
   :environment: MM_SERVICESETTINGS_ENABLESVGS
 
-  - **true**: **(Default)** Enables previews of SVG files attached to messages.
-  - **false**: Disables previews of SVG files.
+  - **true**: Enables previews of SVG files attached to messages.
+  - **false**: **(Default)** Disables previews of SVG files.
 
 Enable SVGs
 ~~~~~~~~~~~
 
 +-------------------------------------------------------------------------------+------------------------------------------------------------------+
-| - **true**: **(Default)** Enables previews of SVG files attached to messages. | - System Config path: **Site Configuration > Posts**             |
-| - **false**: Disables previews of SVG files.                                  | - ``config.json`` setting: ``.ServiceSettings.EnableSVGs: true`` |
+| - **true**: Enables previews of SVG files attached to messages.               | - System Config path: **Site Configuration > Posts**             |
+| - **false**: **(Default)** Disables previews of SVG files.                    | - ``config.json`` setting: ``.ServiceSettings.EnableSVGs: true`` |
```

Comment:
Can you please update the example in the right column from ``true`` to ``false`` as well? Thanks!